tests/causal_refuters/base.py

```diff
@@ -4,13 +4,14 @@
 
 class TestRefuter(object):
     def __init__(self, error_tolerance, estimator_method, refuter_method,
-            outcome_function=None, confounders_effect_on_t=None,
+            outcome_function=None, params = None, confounders_effect_on_t=None, 
             confounders_effect_on_y=None, effect_strength_on_t=None,
             effect_strength_on_y=None, **kwargs):
         self._error_tolerance = error_tolerance
         self.estimator_method = estimator_method
         self.refuter_method = refuter_method
         self.outcome_function = outcome_function
```

Comment:
is outcome_function ever passed?

Reply:
We pass outcome function either when we want to test a user defined study. The other case in which we make use of this variable is when we wish to specify a predefined function such as a neural network to estimate the value of the outcome function.